### DISCUSSION TITLE
Suppressing warnings during open of the font

### DISCUSSION THREAD

**Discussion by ml16685** [fontforge/fontforge#4817] [Q&A]
Hello,

When I open the font using FontForge Python API 

font = fontforge.open(font_file)

there is a number of warnings in the output:

_Warning: Mac string is a subset of the Windows string in the 'name' table
 for the Designer string in the English (US) language._

Could you please tell in more details what do these warnings mean, and if it is not an error,
how to correctly suppress them in the script?

**Comment by WinkelCode** [comment]
Hi, I am looking into the same issue right now. I am working on a script to convert fonts and I can safely discard the warnings. I wish there was a "quiet" toggle for the `open()` function.

**Comment by MechDR** [comment]
Has this been resolved? Is there a way to suppress the warnings?
